packages/pangea-node-sdk/src/utils/multipart.ts

```diff
@@ -0,0 +1,166 @@
+type Part = {
+  contentDispositionHeader: string;
+  contentTypeHeader: string;
+  part: number[];
+};
+
+type Input = {
+  filename: string;
+  name?: string;
+  type: string;
+  data: Buffer;
+};
+
+enum ParsingState {
+  INIT,
+  READING_HEADERS,
+  READING_DATA,
+  READING_PART_SEPARATOR,
+}
+
+export function parse(multipartBodyBuffer: Buffer, boundary: string): Input[] {
+  let lastline = "";
```

Comment:
camelCase like the rest?
```suggestion
  let lastLine = "";
```